WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -321,14 +322,26 @@ public void gotXmlRpcEndpoint(String inputSiteAddress, String endpointAddress) {
     }
 
     private void launchHelpshift(String url, String username, boolean isWpcom) {
-        HelpshiftHelper.Tag origin = HelpshiftHelper.chooseHelpshiftLoginTag(getLoginMode() == LoginMode.JETPACK_STATS,
-                isWpcom);
+        HelpshiftHelper.Tag origin;
+        Fragment currentFragment = getSupportFragmentManager().findFragmentById(R.id.fragment_container);
```

Comment:
Hey @aerych I wonder, any particular reason for not using the already dedicated `helpXXXX()` methods and pass the origin into `launchHelpshift()` from them? The screen fragments already call a unique help callback each so, the origin can is known just by which method is called.

Unless there is some insight I'm missing, I would prefer we use the `helpXXXX()` methods to define the origin as those methods are exactly designed to be screen specific.